src/components/forms/itemDetail/ItemDetailForm.tsx

```diff
@@ -0,0 +1,231 @@
+                    render={({ field: { onChange, value } }) => (
+                      <DatePick
+                        // @ts-ignore
+                        currentDate={value}
+                        onChange={onChange}
+                      />
+                    )}
+                    name="manufacturing_date"
+                  />
+                </div>
+                <div className="flex flex-col gap-2 w-1/3">
+                  <p className="text-xs text-transit-black-secondary font-medium">{format('item_details.received_date.label')}</p>
+                  <Controller
+                    control={control}
+                    render={({ field: { onChange, value } }) => (
+                      <DatePick
+                        // @ts-ignore
+                        currentDate={value}
+                        onChange={onChange}
+                      />
+                    )}
+                    name="received_date"
+                  />
+                </div>
+              </div>
+              <div className="flex flex-row gap-2">
+                <div className="flex flex-col gap-2 w-1/3">
+                  <p className="text-xs text-transit-black-secondary font-medium">{format('item_details.expiry_date.label')}</p>
+                  <Controller
+                    control={control}
+                    render={({ field: { onChange, value } }) => (
+                      <DatePick
+                        // @ts-ignore
+                        currentDate={value}
+                        onChange={onChange}
+                      />
+                    )}
+                    name="expiry_date"
+                  />
+                </div>
+                <div className="flex flex-col gap-2 w-1/3">
+                  <p className="text-xs text-transit-black-secondary font-medium required-field">{format('item_details.batch_namber.label')}</p>
+                  <Input
+                    // eslint-disable-next-line react/jsx-props-no-spreading
+                    {...register('batch_number', {
+                      required: format('validation.error.field_required'),
+                      pattern: {
+                        value: /\d+/,
+                        message: format('validation.error.field_number'),
+                      },
+                      maxLength: {
+                        value: 10,
+                        message: format('validation.error.character_length'),
+                      },
+                    })}
+                    name="batch_number"
+                    id="floatingInput"
+                    placeholder={format('item_details.batch_namber.label')}
+                    type="text"
+                    className="h-9 placeholder-grey-300"
+                    isInvalid={Boolean(errors.batch_number)}
+                  />
+                  {/* @ts-ignore */}
+                  {errors.batch_number && <ValidationError value={errors.batch_number.message} />}
```

Comment:
@KarolinaKopacz is this working for another validations errors you defined while registration of this Input ? (field_number and character length errors)